patch-package
--- a/node_modules/apollo-client/bundle.umd.js
+++ b/node_modules/apollo-client/bundle.umd.js
@@ -128,7 +128,7 @@
                 _this.message = errorMessage;
             }
             _this.extraInfo = extraInfo;
-            Object.setPrototypeOf(_this, ApolloError.prototype);
+            _this.__proto__ = ApolloError.prototype;
             return _this;
         }
         return ApolloError;
